gdb-server/src/target/base.rs

```diff
@@ -21,7 +21,12 @@ impl MultiThreadBase for RuntimeTarget<'_> {
         let mut core = session.core(tid.get() - 1).into_target_result()?;
 
         regs.pc = core
-            .read_core_reg(core.registers().program_counter())
+            .read_core_reg(
+                core.registers()
+                    .program_counter()
+                    .into_target_result()?
```

Comment:
Why is this required?

Reply:
@Yatekii It is used elsewhere in the code also, wherever it is necessary to massage `probe_rs::error::Error` into a `gdb_stub::target::TargetResult`. Now that I typed that, I think I don't really understand your question. Do you mind elaborating please?

Reply:
Ah, silly me, this is GDB stuffs, I must have been sleeping ...